sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -468,6 +461,12 @@ def _get_bigquery_client():
     FROM {{ left_table_query_string }}
 ),
 
+timestamp_bounds AS (
+    SELECT
+        MAX({{entity_df_event_timestamp_col}}) AS max,
+        MIN({{entity_df_event_timestamp_col}}) AS min
```

Comment:
can you call these columns something different from `min` and `max` to avoid confusion between columns and keywords?

Reply:
Done, columns renamed and also moved the timestamp inference into a separate query to avoid recomputing and avoid adding complexity to the main retrieval query.